README.adoc

```diff
@@ -180,9 +180,10 @@ in `InventoryManager.java`. You will look into this class in more detail in the
 
 === Adding the @Retry and @Fallback annotations
 
-Our `inventory` microservice is now able to recognise that the `system` microservice has been taken
-down for maintenance through the 503 HTTP response returned. As a result, it has thrown an IOException.
-A fallback method now needs to be set out to deal with this failure.
+Because of the 503 HTTP response that was returned,
+the `inventory` microservice is now able to recognise that the `system` microservice was taken
+down for maintenance. As a result, the response throws an IOException.
```

Comment:
As a result, the response throws an IOException.
=> This method throws an IOException to simulate system being unavailable.

Reply:
I would just add "the" before "system."